### DISCUSSION TITLE
Debounce still not working on peek

### DISCUSSION THREAD

**Discussion by LorenzoSapora** [laravel/nova-issues#5179] [Q&A] [answered]
This issue ( https://github.com/laravel/nova-issues/issues/4933 ) was closed 2 months ago, but nothing has been mentioned in the changelog ( https://nova.laravel.com/releases ), nor has the issue been resolved. 

Slipped through the cracks?

**Comment by crynobone** [comment, marked as answer]
Submit a proper bug report (with your own code example and use case) if you believe the issue still exists.

**Reply by LorenzoSapora** [reply]
It's exactly the same issue as #4933, but I can't reopen that. Duplicating issues seems like the wrong move.

Peeking still allows someone to move the mouse over a list of peekable relations without a debounce.

**Reply by crynobone** [reply]
We solve those issue, and don't know yours. So create a new bug report with full reproducing code.
